src/services/web3/web3.plugin.ts

```diff
@@ -13,6 +13,8 @@ import { WalletLinkConnector } from './connectors/walletlink/walletlink.connecto
 import { PortisConnector } from './connectors/portis/portis.connector';
 import useFathom from '@/composables/useFathom';
 import getProvider from '@/lib/utils/provider';
+import ConfigService from '../config/config.service';
```

Comment:
If you merge in `develop` you should be able to do this:
```suggestion
import { configService } from '../config/config.service';
```
And then not have to initiate it with `new` below.